functional_tests/test_db_online_offline.py

```diff
@@ -327,24 +327,20 @@ def test_online_to_offline_changes_feed_controlled_close_continuous(cluster, con
         futures[executor.submit(admin.take_db_offline, "db")] = "db_offline_task"
 
         for future in concurrent.futures.as_completed(futures):
-            try:
-                task_name = futures[future]
-
-                if task_name == "db_offline_task":
-                    log.info("DB OFFLINE")
-                    # make sure db_offline returns 200
-                    assert(future.result() == 200)
-                elif task_name == "docs_push":
-                    log.info("DONE PUSHING DOCS")
-                    doc_add_errors = future.result()
-                elif task_name == "continuous":
-                    docs_in_changes = future.result()
-                    log.info("DOCS FROM CHANGES")
-                    for k, v in docs_in_changes.items():
-                        log.info("DFC -> {}:{}".format(k, v))
-
-            except Exception as e:
-                print("Futures: error: {}".format(e))
+            task_name = futures[future]
+
+            if task_name == "db_offline_task":
+                log.info("DB OFFLINE")
+                # make sure db_offline returns 200
+                assert(future.result() == 200)
+            elif task_name == "docs_push":
+                log.info("DONE PUSHING DOCS")
+                doc_add_errors = future.result()
+            elif task_name == "continuous":
+                docs_in_changes = future.result()
+                log.info("DOCS FROM CHANGES")
```

Comment:
I would like to keep most of these as info, just so in the test output, I have an idea of what tasks completed. I will update the printing of docs to debug though.